pdfminer/pdfdocument.py

```diff
@@ -812,3 +816,24 @@ def read_xref_from(self, parser, start, xrefs):
             pos = int_value(trailer['Prev'])
             self.read_xref_from(parser, pos, xrefs)
         return
+    
+    # get AcroForm data
+    def get_acroform(self):
```

Comment:
I meant to extract the method and use it in the docs. People that copy-paste a method are better of than people than people that copy-paste a script. (At least, that's my intuition). 

The method looks good though :) Perhaps we can add it to the `highlevel` methods and rename it to `extract_form_data()`.

Reply:
Oh, I misunderstood.
I'll remove the method from pdfdocument, and change the howto.
It won't take muck for me to make it again for the highlevel later on.

Reply:
I see the pdfdocument in my fork is different (there was a release): what do I need to do?